### DISCUSSION TITLE
View request body and header after such data was sent

### DISCUSSION THREAD

**Discussion by skyffx** [thunderclient/thunder-client-support#926] [Ideas]
Hi, great library, thanks for developing.

Is there a way to view request body and header after such data was sent? The idea is to check what kind of dynamic data is generated in for example body JSON payload for instance `"idNumber": "test-{{#date, 'X'}}"`.

**Comment by rangav** [comment]
@skyffx you can view in the logs

https://github.com/rangav/thunder-client-support#logs

**Reply by skyffx** [reply]
Thank you!
